packages/k8s/k3s-systemd/build.yaml

```diff
@@ -19,27 +19,15 @@ env:
   {{ end }}
 steps:
   - curl -sfL https://get.k3s.io > installer.sh
-  # Let the installer script install service files both for openrc and systemd:
+  # Let the installer script install service files for systemd:
   # https://github.com/k3s-io/k3s/blob/36645e7311e9bdbbf2adb79ecd8bd68556bc86f6/install.sh#L114-L122
   - touch /bin/systemctl && chmod +x /bin/systemctl
   - mkdir -p /etc/systemd/system/
   - bash installer.sh
   - bash installer.sh agent
```

Comment:
you can reuse this and use templating to execute the below commands only if the package is called k3s-openrc or k3s-systemd